assets/extensions/examples/online-extensions.css

```diff
@@ -0,0 +1,6 @@
+/*
+  Customize the logo in the homepage header
+*/
+.navbar-pf-alt #header-logo {
```

Comment:
Our existing style has that selector, so just #header-logo would be a less specific selector.  We could probably update our style to fix that.